benchmark/harness_main.go

```diff
@@ -37,7 +37,7 @@ func DriverBenchmarkMain() int {
 		output = append(output, evg...)
 	}
 
-	evgOutput, err := json.MarshalIndent(map[string]interface{}{"results": output}, "", "   ")
```

Comment:
We don't need the top-level `"results"` key with the new format.

Reply:
Curiously, [The Legacy Data Migration](https://github.com/10gen/performance-tooling-docs/blob/main/getting_started/performance_monitoring_setup.md#legacy-data-migration) docs show a top-level `"data"` field wrapping `"results"`:

```
{
  "data": {
    "results": [
      {
        "name": "insertOne", // test_name
        "results": {
          "16": { // 16 is the thread level, you can't record other args (e.g. compression)
            "ops_per_sec": 10000, // metric, legacy tooling doesn't support anything other than "ops_per_sec"
          },
          "8": {
            "ops_per_sec": 3000,
          }
        },
      },
    ],
  },
}
```

But the [C driver performance tests](https://github.com/mongodb/mongo-c-driver-performance/blob/fc5c741e36e330e8f39989e6c0da3c57e45ca214/src/mongo-c-performance.c#L297:L305) do not include `"data"` either. I am assuming `"data"` was not necessary.